modules/ROOT/pages/installation.adoc

```diff
@@ -7,3 +7,472 @@ endif::[]
 Install API Community Manager with these instructions.
 
 Installed as a Salesforce package.
+
+API Community Manager 
+Setup Guide
+ (RC Version)
+
+Organization Installation Requirements - prereqs???
+Installing API Community Manager 
+Update API Community Manager - move this???
```

Comment:
this should go on a new page `updating.adoc` (I've added the link to the nav doc in the rigth place but haven't created the page)

Reply:
Removed it - thanks!